### PR TITLE
Feat/launch bar icons style

### PR DESCRIPTION
**Pull request by fsarda**
Change to launchbar app:

image
Look and feel changes

image

image

Adding tooltip with app name

### CODE REVIEW

**Comment by rikoe** [comment]
Could you please add some screenshots/gifs @fsarda

**Comment by zhouyuhe** [comment]
image
I think the tooltip should be inline with the bottom of the launch bar

**Comment by mozmorris** [comment]
https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/node-server/openfin-config/src/config/launcher.ts will also need updating 👍

File: src/client/public/config/openfin/launcher-dev.json [hunk 1]
```diff
@@ -39,7 +40,7 @@
     }
   },
   "runtime": {
-    "arguments": "--remote-debugging-port=9222",
+    "arguments": "--remote-debugging-port=9222 --disable-legacy-window",
```
Comment (rikoe) [inline src/client/public/config/openfin/launcher-dev.json]: Why did we add this switch?

File: src/client/public/config/openfin/launcher-demo.json [hunk 2]
```diff
@@ -3,11 +3,12 @@
     "name": "Reactive Ecosystem Launcher",
     "url": "https://web-demo.adaptivecluster.com/launcher",
     "uuid": "reactive-launcher-demo",
-    "defaultWidth": 650,
-    "defaultHeight": 52,
+    "defaultWidth": 400,
+    "defaultHeight": 120,
```
Comment (rikoe) [inline src/client/public/config/openfin/launcher-demo.json]: I wouldn't actually change these JSON files in this way. Actual installed applications without these changes point to this file.

File: src/client/public/config/openfin/launcher-dev.json (hunk 1 above)
Comment (fsarda) [reply src/client/public/config/openfin/launcher-dev.json]: In order to make window transparent (referring to this docs https://developers.openfin.co/docs/faq#can-i-make-a-transparent-window-in-openfin) 

image

File: src/client/public/config/openfin/launcher-demo.json (hunk 2 above)
Comment (rikoe) [reply src/client/public/config/openfin/launcher-demo.json]: The right place to change the config is here: https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/node-server/openfin-config/src/config/launcher.ts

As this config is now served as part of the environment.

Unfortunately the local config does not follow this pattern yet, so please change only launcher-local.json, and the config in the location above.

Comment (rikoe) [reply src/client/public/config/openfin/launcher-demo.json]: The above comments apply to all the configs for dev, uat, demo etc.

**Pull request merged**